src/components/@shared/Onboarding/HideButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from 'react'
+import Button from '../atoms/Button'
+import styles from './HideButton.module.css'
+import Tooltip from '../atoms/Tooltip'
+import Markdown from '../Markdown'
+import Cross from '@images/x-cross.svg'
+
+export default function HideButton({
+  setShowOnboardingModule,
+  hintText
```

Comment:
suggestion(blocking): the same goes for the hint text in this case: you can import it from content directly in this component

Reply:
comment/hint: this wont be possible always ofc, but in general it is a good idea to try and keep components independent and self contained to ensure easier maintainability and re-usability of components